server/test/app.test.js

```diff
@@ -1,53 +1,50 @@
-import { match } from 'assert'
+import test from 'node:test'
+import { equal, match } from 'assert'
```

Comment:
```suggestion
import { equal, match } from 'node:assert'
```

If we use `node:` prefix, we should use it everywhere.